src/sentry/web/urls.py

```diff
@@ -437,13 +437,6 @@ def init_all_applications():
         react_page_view,
         name='sentry-organization-stats'
     ),
-    url(
-        r'^organizations/(?P<organization_slug>[\w_-]+)/teams/new/$', react_page_view),
```

Comment:
This URL has an acceptance test. Can it be removed as well? https://github.com/getsentry/sentry/blob/b1a335a150cfbc17537e0da256becf9099d5238e/tests/acceptance/test_create_team.py#L17